db/models.py

```diff
@@ -1,5 +1,3 @@
-import datetime
-
 from django.contrib.auth.models import AbstractUser
 from django.core.exceptions import ValidationError
```

Comment:
cannot highlight the correct code but
instead of
`import settings`
use
`from django.conf import settings`
because you may have a few settings files or a folder in the same way